modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/SharedTypeScriptTest.java

```diff
@@ -69,4 +71,25 @@ public void oldImportsStillPresentTest() throws IOException {
 
         FileUtils.deleteDirectory(new File("src/test/resources/oldImportsStillPresentTest/"));
     }
+
+    /*
+        #8000
+        Test that primatives are not returned by toModelImportMap
```

Comment:
the comment is a bit too narrow - the actual test not only verifies that, but also checks that  intersection/union type imports are properly handled in general. maybe reword this somehow?